specification/ingest/_types/Processors.ts

```diff
@@ -30,47 +30,233 @@ import { Script } from '@_types/Scripting'
   join?: JoinProcessor
+  /**
+   * Converts a string to its lowercase equivalent.
+   * If the field is an array of strings, all members of the array will be converted.
+   * @doc_id lowercase-processor
+   */
   lowercase?: LowercaseProcessor
+  /**
+   * Removes existing fields.
+   * If one field doesn’t exist, an exception will be thrown.
+   * @doc_id remove-processor
+   */
   remove?: RemoveProcessor
+  /**
+   * Renames an existing field.
+   * If the field doesn’t exist or the new name is already used, an exception will be thrown.
+   * @doc_id rename-processor
+   */
   rename?: RenameProcessor
+  /**
+   * Runs an inline or stored script on incoming documents.
+   * The script runs in the `ingest` context.
+   * @doc_id script-processor
+   */
   script?: Script
+  /**
+   * Sets one field and associates it with the specified value.
+   * If the field already exists, its value will be replaced with the provided one.
+   * @doc_id set-processor
+   */
   set?: SetProcessor
+  /**
+   * Sorts the elements of an array ascending or descending.
+   * Homogeneous arrays of numbers will be sorted numerically, while arrays of strings or heterogeneous arrays of strings + numbers will be sorted lexicographically.
+   * Throws an error when the field is not an array.
+   * @doc_id sort-processor
+   */
   sort?: SortProcessor
+  /**
+   * Splits a field into an array using a separator character.
+   * Only works on string fields.
+   * @doc_id split-processor
+   */
   split?: SplitProcessor
+  /**
+   * Trims whitespace from field.
+   * If the field is an array of strings, all members of the array will be trimmed.
+   * This only works on leading and trailing whitespace.
+   * @doc_id trim-processor
+   */
   trim?: TrimProcessor
+  /**
+   * Converts a string to its uppercase equivalent.
+   * If the field is an array of strings, all members of the array will be converted.
+   * @doc_id uppercase-processor
+   */
   uppercase?: UppercaseProcessor
+  /**
+   * URL-decodes a string.
+   * If the field is an array of strings, all members of the array will be decoded.
+   * @doc_id urldecode-processor
+   */
   urldecode?: UrlDecodeProcessor
+  /**
+   * Converts a human readable byte value (e.g. 1kb) to its value in bytes (e.g. 1024).
+   * If the field is an array of strings, all members of the array will be converted.
+   * Supported human readable units are "b", "kb", "mb", "gb", "tb", "pb" case insensitive.
+   * An error will occur if the field is not a supported format or resultant value exceeds 2^63.
+   * @doc_id bytes-processor
+   */
   bytes?: BytesProcessor
+  /**
+   * Similar to the Grok Processor, dissect also extracts structured fields out of a single text field within a document.
+   * However unlike the Grok Processor, dissect does not use Regular Expressions.
+   * This allows dissect’s syntax to be simple and for some cases faster than the Grok Processor.
+   * Dissect matches a single text field against a defined pattern.
```

Comment:
```suggestion
   * Extracts structured fields out of a single text field by matching the text field against a delimiter-based pattern.
```

Reply:
For aggregations, I've tried as much as possible to add descriptions that do not refer to other aggregations. What would you think of this shorter definition?